molecule/debian/molecule.yml

```diff
@@ -9,7 +9,7 @@ lint: |
 platforms:
   - name: kibana
     hostname: kibana
-    image: ${MOLECULE_DISTRO:-idealista/jdk:11.0.14-buster-openjdk-headless}
+    image: ${MOLECULE_DISTRO:-debian:buster-slim}
```

Comment:
Just as a suggestion, since we're adding support to debian bullseye, maybe we can change molecule to use a bullseye image as well?

Reply:
It's added in travis.yml. 
```   
 - MOLECULE_DISTRO=idealista/jdk:11.0.16-buster-openjdk-headless JAVA_JDK_HOME=/usr/lib/jvm/java-11-openjdk-amd64 KIBANA_VERSION=7.17.10
 - MOLECULE_DISTRO=idealista/jdk:11.0.16-bullseye-openjdk-headless JAVA_JDK_HOME=/usr/lib/jvm/java-11-openjdk-amd64 KIBANA_VERSION=7.17.10
 - MOLECULE_DISTRO=debian:buster-slim KIBANA_VERSION=8.7.1
 - MOLECULE_DISTRO=debian:bullseye-slim KIBANA_VERSION=8.7.1
 ```

Reply:
I'd say it's better to default to bullseye for future tests :)